internal/game/game.go

```diff
@@ -0,0 +1,27 @@
+package game
```

Comment:
I don't think Internal is right for game. I think the goal should be to provide generally useful go libraries -- and certainly Game would be one of those. So, I'd make it a publicly available library at the top level.

Reply:
How about the parser?

Reply:
I'd keep parser public too -- seems like a generally useful tool. Scanner might deserve to be internal, but that could just be private variables within parser

Generally, I feel internal directories should be reserved as a last resort when your private types have grown too unwieldy and you want to share them with other library.

Reply:
Thanks for clarifying, this is super useful for other projects I have at work